sweep/fee_bumper.go

```diff
@@ -444,12 +474,39 @@ func (f *feeBumper) calculateFeeRate(r *monitoredInput) chainfee.SatPerKWeight {
 	case StrategyNone:
 		return r.currentFeeRate
 
+	case StrategyLinear:
+		return f.feeRateForStrategyLinear(r)
+
 	default:
 		feeLog.Errorf("Unknown strategy %v", r.strategy)
 		return r.currentFeeRate
 	}
 }
 
+// TODO(yy): cache the results?
+func (f *feeBumper) feeRateForStrategyLinear(
```

Comment:
If we make this part of the strategy interface, then we can easily add caching there (though seems to be a premature optimization at this point imo).